dist/aframe-extras.controls.js

```diff
@@ -1155,6 +1155,10 @@ module.exports = AFRAME.registerComponent('gamepad-controls', {
   getJoystick: function getJoystick(index, target) {
     var gamepad = this.getGamepad();
     if (gamepad.mapping === 'xr-standard') {
+      var handedness = this.system.controllers[this.data.controller].handedness;
+      if (handedness === 'right') {
+        this.data.controller = 1;
+      }
```

Comment:
Could you omit the changes to `dist/` files from this PR? You can restore the original files with:

 `git checkout master -- dist/`

Reply:
Sure. Done!